src/pages/projects/actions/ProjectUsedBy.tsx

```diff
@@ -1,135 +1,23 @@
 import { type FC } from "react";
 import type { LxdProject } from "types/project";
-import type { LxdUsedBy } from "util/usedBy";
-import { filterUsedByType } from "util/usedBy";
-import { Icon } from "@canonical/react-components";
-import { useImagesInProject } from "context/useImages";
-import ExpandableList from "components/ExpandableList";
-import UsedByItem from "components/UsedByItem";
-
-const INSTANCES = "Instances";
-const PROFILES = "Profiles";
-const IMAGES = "Images";
-const CUSTOM_VOLUMES = "Custom volumes";
+import UsedByRow from "components/UsedByRow";
 
 interface Props {
   project: LxdProject;
 }
 
 const ProjectUsedBy: FC<Props> = ({ project }) => {
-  const { data: imagesFromProject = [] } = useImagesInProject(project.name);
-  const data: Record<string, LxdUsedBy[]> = {
-    [INSTANCES]: filterUsedByType("instance", project.used_by),
-    [PROFILES]: filterUsedByType(
-      "profile",
-      project.used_by?.filter(
-        // the default profile is not blocking project deletion and can't be removed itself
-        (item) => !item.startsWith("/1.0/profiles/default"),
-      ),
-    ),
-    [IMAGES]: filterUsedByType("image", project.used_by),
-    [CUSTOM_VOLUMES]: filterUsedByType("volume", project.used_by),
-  };
+  const usedBy = project.used_by?.filter(
+    (t) => !t.startsWith("/1.0/profiles/default"),
+  );
```

Comment:
```suggestion
  // the default profile is not blocking project deletion and can't be removed itself
  const usedBy = project.used_by?.filter(
    (t) => !t.startsWith("/1.0/profiles/default"),
  );
```
Shall we keep the comment?